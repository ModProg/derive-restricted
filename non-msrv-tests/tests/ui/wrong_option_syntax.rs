#[derive_where::derive_where(Debug = "option"; T)]
struct Test<T>(T);

fn main() {}
