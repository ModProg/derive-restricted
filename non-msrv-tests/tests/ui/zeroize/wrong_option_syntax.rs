#[derive(derive_where::DeriveWhere)]
#[derive_where(Zeroize("option"); T)]
struct Test1<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Zeroize(crate(zeroize_)); T)]
struct Test2<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Zeroize(crate = "struct Test"); T)]
struct Test3<T>(T);

fn main() {}
