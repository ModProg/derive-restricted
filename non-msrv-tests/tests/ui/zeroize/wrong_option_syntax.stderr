error: unexpected option syntax
 --> tests/ui/zeroize/wrong_option_syntax.rs:2:24
  |
2 | #[derive_where(Zeroize(crate(zeroize_)); T)]
  |                        ^^^^^^^^^^^^^^^
